### DISCUSSION TITLE
How to specfiy minecrat Java version?

### DISCUSSION THREAD

**Discussion by Elekam** [GameServerManagers/LinuxGSM#4422] [Q&A]
Hey, I would like to downgrade my 1.20.4 server to 1.20.1. I see a versions folder  in /serverfiles/ and found this [documentation](https://docs.linuxgsm.com/game-servers/minecraft).

It says I can specify the version, but doesn't elaborate how. I cant find it anywhere. Ig I might be able to replace the minecraft_server.jar with a 1.20.1 before install but is that the right way?

Edit: I found the "mcversion" variable in the _default.cfg of lgsm-config. I added this variable to the mcserver.cfg and reinstalled the server. But the version is still 1.20.4. Does it ignore the variable, does it not use the mcserver.cfg?

**Comment by EthanDMello** [comment]
Hey @Elekam,

Sorry to reply on a 2 yo post. But I have the same issue and also couldn't find solution.
Like you've said, https://docs.linuxgsm.com/game-servers/minecraft specifies which releases/snapshots you can use but doesn't specifiy what to do after you add the "mcversion" in common or instance config files.

Turns out you have to use ./gameserver update and it'll use the version specified in your config file, rather than the default value of latest in _default.cfg
